src/main.py

```diff
@@ -1,13 +1,16 @@
-from networking import NetworkingManager
+#from networking import NetworkingManager
 from render import RenderingManager
-from database import DatabaseManager
+#from database import DatabaseManager
+from tkinter import *
+from tkinter import ttk
 
 gameState = {"running": True, "stage": "init"} # global stuff like what players on what team, what is the current state of the game (starting, inprogress, finished) and other stuff like that.
 
-databaseManager = DatabaseManager(gameState)
-networkingManager = NetworkingManager(gameState)
-renderingManager = RenderingManager(gameState, networkingManager, databaseManager)
+#databaseManager = DatabaseManager(gameState)
+#networkingManager = NetworkingManager(gameState)
+renderingManager = RenderingManager(gameState)
+renderingManager.mainloop()
```

Comment:
It seems like this mainloop call blocks the below networking loop/calls from running. Is it possible to add the networking call to this mainloop or render tkinter without using its mainloop, but instead calling the render frame function directly?

Reply:
should be fixed by calling update() now instead of mainloop()